frontend/app/src/router/index.ts

```diff
@@ -466,7 +466,7 @@ router.beforeEach((to, from, next) => {
     if (
       [Routes.USER, Routes.USER_CREATE, Routes.USER_LOGIN].includes(to.path)
     ) {
-      return;
+      next(Routes.DASHBOARD);
     }
     next();
```

Comment:
@lukicenturi will this not mean that `next` will be called twice?

so, either `return next(Routes.DASHBOARD)` or put `next()` in an else.

Reply:
Updated